src/pspm_get_viewpoint.m

```diff
@@ -309,6 +309,18 @@
 end
 
 function import_cell = import_marker_chan(import_cell, markers, mi_names, mi_values, n_rows, sampling_rate)
+    % Put here all characters which do not belong to markers
+    % they have to be separated by a '|'
+    % '=' has a special threatment because we do not want to delete 'CS+'
+    non_markers = [',','|','(?<!\w)(+)','|','='];
+    
+    mi_names = regexprep(mi_names,non_markers,'');
+    non_empty = find(~cellfun('isempty',mi_names));
```

Comment:
This would work, I think. We don't even need the complex regex pattern for `+` since if the marker has at least one character different than `,+=`, it will still be non-empty which is enough. However, it would be better to assign the regex result to a new variable in line 317 instead of `mi_names`. The reason is that currently this can change the name of a marker. For example, if a marker is named `CS=`, this would return only `CS`.

Reply:
As a side note, I noticed a bug in import regarding session beginning offsets. I implemented a quick fix and will submit it once we fix this one.